src/phases/query/IDHack.yml

```diff
@@ -22,6 +23,47 @@ IDHackObjectIdTemplate:
           OperationCommand:
             Filter: {_id: {^ObjectId: {^FormatString: {"format": "%07d%s", "withArgs": [{^RandomInt: {min: 0, max: 9999999}}, "bbbbbbbbbbbbbbbbb"]}}}}
 
+# Runs IDHack find queries, matching an _id of type int.
+IDHackIntIdTemplate:
+  Name: {^Parameter: {Name: "Name", Default: "IDHackIntId"}}
+  Type: CrudActor
+  Threads: {^Parameter: {Name: "Threads", Default: 1}}
+  Phases:
+    OnlyActiveInPhases:
+      Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: {unused: "please specify in which phases this actor should be active."}}}]
+      NopInPhasesUpTo: {^Parameter: {Name: "NopInPhasesUpTo", Default: 99}}
+      PhaseConfig:
+          # Our testing suggested the first ~20 seconds of this actor shows a lot of noise before
+          # stabilizing - 100s should gather sufficient post-noise data.
+        Duration: 100 seconds
+        Database: {^Parameter: {Name: "Database", Default: test}}
+        Collection: {^Parameter: {Name: "Collection", Default: Collection0}}
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            {Filter: {_id: {^RandomInt: {min: 0, max: 9999999}}}}
+
+# Runs IDHack aggregation queries, matching an _id of type int.
+IDHackIntIdAggTemplate:
+  Name: {^Parameter: {Name: "Name", Default: "IDHackIntIdAgg"}}
+  Type: CrudActor
+  Threads: {^Parameter: {Name: "Threads", Default: 1}}
+  Phases:
+    OnlyActiveInPhases:
+      Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: {unused: "please specify in which phases this actor should be active."}}}]
+      NopInPhasesUpTo: {^Parameter: {Name: "NopInPhasesUpTo", Default: 99}}
+      PhaseConfig:
+          # Our testing suggested the first ~20 seconds of this actor shows a lot of noise before
```

Comment:
This is reading as slightly dishonest :D Do you want to double check? or just remove this comment or reword it to say "matching the query timeline" or something?

Reply:
Ah good point. Updated to say `100 seconds to match the find query timeline.`